keras/backend/cntk_backend.py

```diff
@@ -2,6 +2,7 @@
 import cntk as C
 import numpy as np
 from .common import _FLOATX, _EPSILON, image_dim_ordering, image_data_format
+from .common import is_placeholder
```

Comment:
Yes, I think so, how else could it be provided in the backend? Is there a better option?